editor/plugins/curve_editor_plugin.h

```diff
@@ -117,6 +117,7 @@ class CurveEditor : public Control {
 	// Constant
 	float _hover_radius;
 	float _tangents_length;
+	float _gizmo_handle_scale;
```

Comment:
```suggestion
	float _gizmo_handle_scale = 0.0;
```
This should be initialized.
(other values too, but you don't need to change them)